src/Protocol.cpp

```diff
@@ -606,3 +600,100 @@ PeriodicUpdate protocol::parseE4Output(uint8_t const* buffer, int bufferSize)
     result.filter_state = state;
     return result;
 }
+
+PeriodicUpdate protocol::parseE5Output(uint8_t const* buffer, int bufferSize)
+{
+    uint8_t const* end = buffer + bufferSize;
+
+    uint8_t const* cursor = buffer;
+    uint32_t time_ms;
+    cursor = endianness::decode<uint32_t>(buffer, time_ms, end);
+    uint8_t filterFlags = cursor[0];
+    ++cursor;
+    int8_t board_temperature_C = *reinterpret_cast<int8_t const*>(cursor);
+    ++cursor;
```

Comment:
What do you think about 
```suggestion
    uint8_t filterFlags = *cursor++;
    int8_t board_temperature_C = *reinterpret_cast<int8_t const*>(cursor++);
```

Reply:
I don't like suffix++, because it requires to know and parse precedence rules. I prefer ++prefixed or being explicit.